docs/flows-format.adoc

```diff
@@ -0,0 +1,250 @@
+// Automatically generated by 'hack/asciidoc-flows-gen.sh'. Do not edit.
+:_content-type: REFERENCE
+[id="network-observability-flows-format-specifications_{context}"]
+= Network Flows format specifications
+
+Description::
++
+This is the specification of the network flows format, used both internally and in exporter feature such as Kafka export.
++
+The document is organized in two main categories: _Labels_ and regular _Fields_. This distinction only matters when querying Loki, as _Labels_, unlike _Fields_, must be used in link:https://grafana.com/docs/loki/latest/logql/log_queries/#log-stream-selector[stream selectors].
++
+Outside of Loki context, for instance when reading this specification as a documentation for the Kafka export feature, you must ignore this distinction about _Labels_ and _Fields_, and consider all of them at the same level in the JSON structure.
+
+
+== Labels
+
+
+=== SrcK8S_Namespace
```

Comment:
I'm getting asciibinder errors with these H3's. I was wondering about making these list items, as you can see here: https://github.com/openshift/openshift-docs/pull/56335/files

wdyt?

Reply:
Sure
Do you see anything else that should be changed? The format is dictated by `typedoc`, so I can't do absolutely anything I want without too much pain, but some little things can certainly be adapted. For instance, I was wondering if the line separators `'''` were really useful or if we should remove them.. wdyt?

Reply:
When looking in my editor renderer, the "+" seem important to make all the description appear at the same level of indentation

Reply:
Ah, I see. Is the 'Description::' a necessary element for your build? I think we can also do without the 'Description::' and leave the text as introductory text that follows the H1 "Network Flows format specifications". 

Another thing to mention about the Description is that I want to relocate this sentence: "This is the specification of the network flows format, used both internally and in exporter feature such as Kafka export" so that it is introductory text for the Assembly file. Like this: 
![image](https://user-images.githubusercontent.com/77501157/222264328-5bc3599b-7805-476f-8781-66554fb427f3.png)

Reply:
As discussed, I have changed the generated file to make it an ASSEMBLY file